lib/common.js

```diff
@@ -25,6 +27,15 @@ export const supportedLocalesDetails = [
 ];
 export const supportedLocales = supportedLocalesDetails.map((l) => l.code);
```

Comment:
Haha these different lists are pretty confusing and I was responsible for making some/all of them! Is it:

- `supportedLocales`: full base + regional codes... not sure what this is actually used for?
- `supportedLocalesDetails`: only used in core demo snippet
- `supportedBaseLocales`: supported locales but just the base part... seems to only be used within Intl
- `supportedLangpacks`: are these the full langpack codes coming out of Brightspace?